metaworld/policies/sawyer_pick_place_v2_policy.py

```diff
@@ -32,19 +32,16 @@ def get_action(self, obs):
     @staticmethod
     def _desired_pos(o_d):
         pos_curr = o_d['hand_pos']
-        pos_puck = o_d['puck_pos']
+        pos_puck = o_d['puck_pos'] + np.array([-0.005, 0, 0])
```

Comment:
.005 is a really small offset. I guess I shouldn't be surprised that something so small can make a difference, but just double check that it's actually necessary.

Reply:
Sadly it does. I removed it and the policy success rate dropped about 20%. I was trying to adjust the xmls to account for this but wasn't successful so I left it in.